cells/lib/ops/revise.nix

```diff
@@ -0,0 +1,65 @@
+{
+  inputs,
+  cell,
+}: let
+  inherit (inputs) nixpkgs;
+  l = nixpkgs.lib // builtins;
+in
+  /*
+  For use with `revisePackage` and `reviseOCI` to build containers in a mono-repo style
+  environment where the source code is contained in the same repository as the std code,
```

Comment:
I think that is an unnecessary restriction.

This helps whenever unrelated changes should not bump the `rev` of a build.

Reply:
No it really doesn't, this information is only useful for the image tag. The Hash is going to change if the rev changes if the rev is included in the output. The only way to avoid that is to not pass in the revision at all.

Reply:
One use case may be a published installer of something in a monorepo that also needs its most significant revision baked into it.

Reply:
```
Allows to find the most significant commit in a monorepo where the source is filtered, but still needs a revision. For use with `revisePackage` and `reviseOCI`.
```